src/Symfony/Component/CssSelector/XPath/Extension/HtmlExtension.php

```diff
@@ -102,31 +102,31 @@ public function translateEnabled(XPathExpr $xpath): XPathExpr
         return $xpath->addCondition(
             '('
                 .'@href and ('
-                    ."name(.) = 'a'"
-                    ." or name(.) = 'link'"
-                    ." or name(.) = 'area'"
+                ."name(.) = 'a'"
+                ." or name(.) = 'link'"
+                ." or name(.) = 'area'"
                 .')'
-            .') or ('
```

Comment:
original, manual indentation logic cannot be applied here.

maybe Nowdoc/heredoc can be used instead, to keep visual appealing aspect